tests/sub_port_interfaces/conftest.py

```diff
@@ -36,6 +36,14 @@ def pytest_addoption(parser):
         help="Max numbers of sub-ports for test_max_numbers_of_sub_ports test case",
     )
 
+    parser.addoption(
+        "--range_of_parent_port",
+        action="store",
+        type=str,
+        default='1-2',
+        help="Range of parent ports for creating sub-ports",
+    )
+
```

Comment:
It's a good idea to add a ```range_of_partent``` option for running on certain a testbed or topology. But it will cause trouble for automation test since we have various topos.  Normally, we are using a universal template to trigger all tests, which means it's difficult to set different ```range_of_parent_port``` for different testbed.
So I think using some logic in code to select parent port for testing is better, although it's not as flexible.

Reply:
Agree with you. Thanks, for your review.